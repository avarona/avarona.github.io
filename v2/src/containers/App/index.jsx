import React from "react";
import { BrowserRouter as Router, Route } from "react-router-dom";
import { Grommet, Main, Box } from "grommet";
import Projects from "../Projects";
import About from "../About";
import Contact from "../Contact";
import NavBar from "../../components/NavBar";
import Footer from "../../components/Footer";

const theme = {
  global: {
    font: {
      family: "Arial, Helvetica, sans-serif"
    }
  }
};

const App = () => (
  <Router>
    <Grommet theme={theme} full>
      <Main>
        <NavBar />
        <Box flex>
          <Route path='/projects' component={Projects} />
          <Route path='/about' component={About} />
          <Route path='/contact' component={Contact} />
          <Route exact path='/' component={Projects} />
        </Box>
        <Footer />
      </Main>
    </Grommet>
  </Router>
);

export default App;
