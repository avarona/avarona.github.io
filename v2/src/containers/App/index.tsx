import * as React from 'react';
import logo from 'images/logo.svg';
import './styles.css';

import Button from 'components/Button';

const App = () => (
  <div className="App">
    <header className="App-header">
    </header>
    <Button svgIcon={logo} text="Submit" />
  </div>
);

export default App;
