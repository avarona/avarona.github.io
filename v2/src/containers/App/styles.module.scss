body {
  margin: 0;
  font-family: Arial, Helvetica, sans-serif;
}

.container {
  width: 100%;
  height: 150vh;
}
