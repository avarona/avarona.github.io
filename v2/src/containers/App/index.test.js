import React from "react";
import { shallow } from "enzyme";
import App from "./";

describe("App Container", () => {
  it("renders", () => {
    const wrapper = shallow(<App />);
    expect(wrapper.html()).toMatchSnapshot();
  });
});
