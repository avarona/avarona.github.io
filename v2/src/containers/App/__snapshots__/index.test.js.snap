// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App Container renders 1`] = `"<div class=\\"container\\"><img src=\\"logo.svg\\" alt=\\"react logo\\"/></div>"`;
